src/inputs/Integer.tsx

```diff
@@ -25,9 +37,17 @@ const IntegerInput = (
   }
   return (
       <div className="input-group integer-input">
-        <label htmlFor={name} title={title}>
-          {label}
-        </label>
+        <div className='label-wrapper'>
+          <label htmlFor={name} title={title}>
+            {label}
+          </label>
+          {
```

Comment:
can this shared code be in a component?

Reply:
I assume you mean the two `withTooltip` ternaries? I'll give it a go.